README.md

```diff
@@ -19,23 +19,24 @@ open-source code, distributed freely under the terms of the GNU Public
 License (GPL).
 
 The maintainer of Pizza.py is Steve Plimpton, who can be emailed at
-sjplimp@sandia.gov.  The Pizza.py WWW site
-(www.cs.sandia.gov/~sjplimp/lammps.html) and LAMMPS WWW Site
-(www.cs.sandia.gov/~sjplimp/lammps.html) have more information about
+sjplimp@sandia.gov.  The Pizza.py [WWW site](https://pizza.sandia.gov/)
```

Comment:
I changed this file to have an `.md` extension. GitHub automatically detects this and formats it accordingly.